python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -209,7 +209,7 @@ def preprocess_ondisk_dataset(input_config_path: str) -> str:
     with open(output_config_path, "w") as f:
         yaml.dump(output_config, f)
     print("Finish preprocessing the on-disk dataset.")
-    return output_config_path
+    return str(output_config_path)
```

Comment:
this only print the path of yaml file as string, but does not print the paths within the yaml file content. Please add dedicated test to make sure all the paths in yaml file is printed as raw string instead of `Path` object.

Reply:
I understand what you mean, I have now changed all `Path` to `str`, but I have a question: why the paths I saved before are all normal.